test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/ExportsTest.scala

```diff
@@ -270,6 +270,10 @@ class ExportsTest {
     assertThrows(classOf[js.JavaScriptException], {
       x.bar = 2
     })
+
+    // Read the properties to trick GCC into not discarding the writes.
+    assertEquals(x.foo, 1)
+    assertEquals(x.bar, 1)
```

Comment:
The expected value is supposed to be on the left:
```suggestion
    assertEquals(1, x.foo)
    assertEquals(1, x.bar)
```